WordPress/openverse-api
# Update Terms of Service

## Description


Follow up from #662

This PR updates the Terms of Service document for the Openverse API. The current version of the document is hosted here: https://api.openverse.engineering/terms_of_service.html and can be found in code here: https://github.com/WordPress/openverse-api/blob/v2.4.3/api/terms_of_service.html

The new ToS will be located along with the rest of our API dev documentation here: https://wordpress.github.io/openverse-api/terms_of_service.html (URL won't work until this is merged).

## :warning: ToS changes :warning:

This PR also includes changes to our ToS wording. A rough difference view between the two can be found here: https://www.diffchecker.com/gc9T5FrF

**Most notably**, the following has changed:
- References to "CC Catalog" and "CC API` have been changed to reference Openverse.
- References to the ["Creative Commons Master Terms of Use"](https://creativecommons.org/terms/) have been removed. As far as I could tell, WordPress Foundation did not have an equivalent.
- Replaced reference to "Openverse Trademark Policy" (likely was "Creative Commons Trademark Policy) with references to WordPress Foundation's Trademark Policy
- Various spacing changes

## Testing Instructions

1. `just sphinx-make`
1. `just sphinx-serve`
1. Change Line 17 in `swagger.py` to be `tos_url = "http://0.0.0.0:3001/terms_of_service.html"`
1. Visit http://localhost:8000 and click the "Terms of Service" link
1. Verify that you are indeed redirected to the locally served sphinx documentation

## Checklist


[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
